MetadataExtractor.PowerShell/ExtractMetadataCmdlet.cs

```diff
@@ -23,7 +24,7 @@ protected override void ProcessRecord()
         {
             base.ProcessRecord();
 
-            WriteVerbose($"Extracting metadata from file: {FilePath}");
+            WriteVerbose(string.Format(CultureInfo.InvariantCulture, "Extracting metadata from file: {0}", FilePath));
```

Comment:
Pardon my ignorance, but how can culture influence string concatenation? For example, other cases were previously `$"Foo {nameof(Bar)}", which I believe the compiler is within rights to optimise to `"Foo Bar"` at compile time.

Reply:
String interpolation uses the current culture. There are issues for cases like
```csharp
var bar = 1.23;
$"Foo {bar}"
```
but (and you are right) not for cases like:
```csharp
var bar = 1.23;
$"Foo {nameof(bar)}"
```
If you like I revert the changes for the cases with `nameof`.

Reply:
Finally also cases like
```csharp
var bar = 1.23;
string.Concat("Foo ", bar);
"Foo " + bar;
```
are not taking culture into account.

Reply:
Did you only fix warnings produced by the analyzer, or did you search for all string interpolation?

I don't see warnings when reverting some of these changes. Where formattable types are involved, fair enough, but many of these changes involve only strings and the result is a reduction in both readability and performance.

https://sharplab.io/#v2:D4AQjAbABCBMUGECwAoA3qqWYGYaXwAYoBZMACgEsA7AFykoEooBeAPigBIAiAQQCEEUNJQC+UACIBRAGLcA3Jmwg84aOGIlY5DVADOzdlz6Dhe8dLmKU2XPnVhNOHY/2GOPAULTUAhgFsAUwB7ADNyAwtZBSUsWLs1IlIAFio6BnciADoZYIAnf19achNvQii5ABoM61sVeySSAFYXYgNWDg0c/MLi0uFyyWjqg1rlVQJdEghWtw7s3IKikq8Biu5qvyCwiMZGa1EgA

I should point out that I'm also trying to work out whether the analyzer can be improved in order to provide feedback to that team.

Reply:
Visual studio helped me to find all places. I didn't get warnings. Knowing that interpolated strings use the current culture I tried to find them all. Validation was done with `git grep`.

I found another notation the keep the readability.
```csharp
FormattableString.Invariant($"ABC {i} DEF");
```
Do you like that?